src/reducers/cartReducer.js

```diff
@@ -1,12 +1,81 @@
+import { SET_QUANTITY, REMOVE_ITEM, INCREASE_QUANTITY, DECREASE_QUANTITY } from '../constants/actionTypes';
+
 const initialState = {
-  items: []
+  items: [
+    {
+      id: 'EOK3293KSD',
+      name: "Men's Chaps Performance Series Classic-Fit 4-Way Strech Sit Jacket",
+      imgUrl: 'https://images-na.ssl-images-amazon.com/images/I/71ZgPluVoTL._UX385_.jpg',
+      variants: {
+        size: '42 SHORT',
+        color: 'Charcoal'
+      },
+      price: {
+        regular: 240,
+        discount: 125
+      },
+      quantity: 1
+    },
+    {
+      id: 'EOK32943983',
+      name: "WoMen's Chaps Performance Series Classic-Fit 4-Way Strech Sit Jacket",
+      imgUrl: 'https://n.nordstrommedia.com/ImageGallery/store/product/Zoom/6/_103433446.jpg',
+      variants: {
+        size: '42 SHORT',
+        color: 'Charcoal'
+      },
+      price: {
+        regular: 440,
+        discount: 140
+      },
+      quantity: 1
+    }
+  ]
+};
+
+const getItemQuantity = function(array, itemId) {
```

Comment:
I am not a big fan of `quantity`. Why not a simpler `amount`? `getItemAmount`, `increaseItemAmount`, `setItemAmount`, `item.amount`.

Reply:
Imho quantity is commonly used in ecommerce, eg. zalando 
![image](https://user-images.githubusercontent.com/45630066/55546647-0e9f0680-56d0-11e9-88f3-de7bd15d84b6.png)
PS. https://www.bkacontent.com/commonly-confused-words-amount-number-and-quantity/ :P